<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Taylor Series Calculator</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout" stretch="0,0,0,1,1">
    <item>
     <widget class="QLineEdit" name="xInput">
      <property name="placeholderText">
       <string>Enter value of x</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="functionCombo">
      <item>
       <property name="text">
        <string>sin</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>cos</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>tan</string>
       </property>
      </item>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="angleCombo">
      <item>
       <property name="text">
        <string>radians</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>degrees</string>
       </property>
      </item>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="calculateButton">
      <property name="text">
       <string>Calculate</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="resultLabel">
      <property name="text">
       <string>Result:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="termsLabel">
      <property name="text">
       <string>Taylor Series:</string>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
